parsers/s01-parse/hitech95/nginx-mail-logs.md

```diff
@@ -0,0 +1,2 @@
+A generic parser for nginx mail module, support both access and error logs.
+
```

Comment:
Can you add an example of an acquisition configuration for this parser please?

Reply:
I have no idea on how to do this... I'm still trying to understand how the acquis file is used to match with a parser.

Reply:
see above conversation, docs is not 100% clear on how the mapping between logs/acquis and parsers/collections are done.